README.md

```diff
@@ -1,103 +1,85 @@
 
-[![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
-[![JetBrains plugins][plugin-version-svg]][plugin-repo]
-![Downloads](https://img.shields.io/jetbrains/plugin/d/14856-liberty-tools)
+**Note: This is an early release.**
 
-A [Liberty](https://openliberty.io/) extension for IntelliJ IDEA. The extension will detect your Liberty Maven or
-Liberty Gradle project if it detects the `io.openliberty.tools:liberty-maven-plugin` in the `pom.xml` or
-`io.openliberty.tools:liberty-gradle-plugin` in the `build.gradle`. Through the Liberty tool window, you can start,
-stop, or interact with [Liberty dev mode](https://openliberty.io/docs/latest/development-mode.html) on all configured
-[Liberty Maven](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev)
-and [Liberty Gradle](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md) projects in your
-workspace.
+![Liberty Tools Extension](docs/images/liberty-tool-window-view.png)
 
-Note that this extension requires the [Integrated Terminal plugin](https://plugins.jetbrains.com/plugin/13123-terminal)
-to be enabled.
+This plugin allows you to run your Liberty Maven or Liberty Gradle projects through the Liberty tool window or the IntelliJ "Go to Action" menu. You can start, stop, or interact with [Liberty dev mode](https://openliberty.io/docs/latest/development-mode.html) on all configured [Liberty Maven](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev) and [Liberty Gradle](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md) projects in your workspace. Liberty Tools help you quickly and easily edit your application and configuration files by providing language support features for MicroProfile, Jakarta EE, and Liberty configuration and Java files.
 
-![Liberty Tools Extension](docs/images/open-liberty-tools.png)
+## Features
+- View Liberty projects in the Liberty tool window.
+- Start/Stop dev mode.
+- Start dev mode with custom parameters.
+- Start dev mode in a container.
+- Start dev mode with the debugger attached.
+- Run tests.
+- View test reports.
+- Editing assistance for Liberty configuration files: server.xml, server.env, bootstrap.properties.
+- Editing assistance for Jakarta EE APIs in Java files.
+- Editing assistance for MicroProfile APIs in microprofile-config.properties and Java files.
+
+<figure>
+  <img
+  src="docs/images/LCLS-server-xml-completion.png"
+  alt="Liberty Config Language Server completion">
+  <figcaption align = "center">Feature completion when editing server.xml files</figcaption>
```

Comment:
sentence suggestion:

Code completion in the server.xml files helps you quickly edit your Liberty runtime configuration.